ipatests/test_integration/test_installation.py

```diff
@@ -388,6 +389,22 @@ def test_install_kra(self):
     def test_install_dns(self):
         tasks.install_dns(self.master)
 
+    def test_WSGI_worker_process(self):
+        """ Test if WSGI worker process count is set to 4
+
+        related ticket : https://pagure.io/freeipa/issue/7587
+        """
+        # check process count in httpd conf file
+        exp_str = b'WSGIDaemonProcess ipa processes={}'.format(
```

Comment:
In python3 this will raise an Exception (AttributeError: 'bytes' object has no attribute 'format')